boxes.lisp

```diff
@@ -537,3 +540,39 @@ N.B., Successive calls may return the same object."
 
 (defmethod baseline ((box sqrt-box))
   (baseline (sqrt-box-base box)))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;; COLUMN BOXES ;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defclass column-box (box)
+  ((padding :initarg :padding
+            :type (integer 0)
+            :accessor column-box-padding)
+   (align :initarg :align
+	  :accessor column-box-align)
+   (contents :initarg :contents
+             :accessor column-box-contents))
+  (:documentation "A vertical concatenation of boxes."))
+
+(defun column-box (boxes &key (padding 0) (align 'left))
```

Comment:
See previous comment about alignment options as keywords. This would be a different type, `horizontal-alignment`.